### PR TITLE
Apply sort node on save when resx file is changed

### PR DESCRIPTION
**Pull request by charles7668**

### CODE REVIEW

File: src/ResXManager.View/Tools/ResXFileWatcher.cs [hunk 1]
```diff
@@ -65,6 +65,9 @@ protected override void OnFilesChanged()
                 if (language.HasChanges)
                     continue;
 
+                if(_resourceManager.Configuration.SortFileContentOnSave)
```
Comment (tom-englert) [inline src/ResXManager.View/Tools/ResXFileWatcher.cs:68]: - Please stick to projects formatting rules (white-space, braces)
- code dup: dereferencing config three times

**Review by tom-englert — CHANGES_REQUESTED** [review]

File: src/ResXManager.View/Tools/ResXFileWatcher.cs [hunk 2]
```diff
@@ -65,6 +65,13 @@ protected override void OnFilesChanged()
                 if (language.HasChanges)
                     continue;
 
+                var config = _resourceManager.Configuration;
+                var sortFileContentOnSave = config.SortFileContentOnSave;
```
Comment (tom-englert) [inline src/ResXManager.View/Tools/ResXFileWatcher.cs:69]: Now that's a bit overkill

File: src/ResXManager.View/Tools/ResXFileWatcher.cs [hunk 3]
```diff
@@ -65,6 +65,13 @@ protected override void OnFilesChanged()
                 if (language.HasChanges)
                     continue;
 
+                var config = _resourceManager.Configuration;
+                var sortFileContentOnSave = config.SortFileContentOnSave;
+                var resxSortingComparison = config.ResXSortingComparison;
+                var duplicateKeyHandling = config.DuplicateKeyHandling;
+                if (sortFileContentOnSave)
+                    language.SortNodes(resxSortingComparison, duplicateKeyHandling);
```
Comment (tom-englert) [inline src/ResXManager.View/Tools/ResXFileWatcher.cs:73]: add braces if its more than a simple statement

**Review by tom-englert — CHANGES_REQUESTED** [review]

File: src/ResXManager.View/Tools/ResXFileWatcher.cs (hunk 3 above)
Comment (tom-englert) [reply src/ResXManager.View/Tools/ResXFileWatcher.cs]: What about this?

Comment (charles7668) [reply src/ResXManager.View/Tools/ResXFileWatcher.cs]: sorry , forgot it , fixed

**Comment by tom-englert** [comment]
👍😀

**Pull request merged**